static/js/util/util.js

```diff
@@ -239,3 +240,26 @@ export function calculateDegreeInclusions(profile: Profile) {
   }
   return inclusions;
 }
+
+type SortableEntry = WorkHistoryEntry|EducationEntry;
+export function resumeOrder(entries: SortableEntry[], dateFieldName: string): SortableEntry[] {
+  let sortFunc = (a, b) => {
```

Comment:
Can you rename `a` and `b` to `start` and `end`, and also the same with `adate` and `bdate`?